backend/substrapp/tasks/utils.py

```diff
@@ -40,8 +39,12 @@ def get_asset_content(channel_name, url, node_id, content_checksum, salt=None):
     return get_remote_file_content(channel_name, url, authenticate_worker(node_id), content_checksum, salt=salt)
 
 
-def get_and_put_asset_content(channel_name, url, node_id, content_checksum, content_dst_path, hash_key):
-    return get_and_put_remote_file_content(channel_name, url, authenticate_worker(node_id), content_checksum,
+def get_and_put_asset_content(channel_name, url, node_id, content_checksum, content_dst_path, hash_key, auth=None):
```

Comment:
Nit: IMO it would be clearer if `auth` was a required parameter

Reply:
This modification was done to handle the particular issue of using `Process` to fetch all models in parallel which seems to mess the usage of Django db model to get OutGoing authentication properly.
For that, we need to get and give the authentication before launching the function in the `Process`, but we may want to keep the old behavior (get authentication in the function) for other call.